homeassistant/components/androidtv/media_player.py

```diff
@@ -192,11 +210,13 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         else:
             device_name = "Android TV / Fire TV device"
 
-        _LOGGER.warning("Could not connect to %s at %s %s", device_name, host, adb_log)
+        _LOGGER.warning(
+            "Could not connect to %s at %s %s", device_name, address, adb_log
+        )
         raise PlatformNotReady
 
-    if host in hass.data[ANDROIDTV_DOMAIN]:
-        _LOGGER.warning("Platform already setup on %s, skipping", host)
+    if address in hass.data[ANDROIDTV_DOMAIN]:
+        _LOGGER.warning("Platform already setup on %s, skipping", address)
```

Comment:
Can't we add a return after this line ?

Reply:
Done. 

Although that might cause the coverage check to fail. I can't seem to hit that block. I tried here and it hits that block when I run it in my dev container (as verified by print statements), but it doesn't show as covered in the pull request coverage report. 

https://github.com/home-assistant/home-assistant/blob/16835bf67cda69fdead58f9a8c092fa4ba5249c2/tests/components/androidtv/test_media_player.py#L574-L586

Reply:
> codecov/patch — 96.07% of diff hit (target 94.49%)

So the coverage check passes. But I'd still like to know how to hit that block of code.

https://github.com/home-assistant/home-assistant/blob/5e9af07ca6385936dea4f72ca4172f06d7a351e5/homeassistant/components/androidtv/media_player.py#L218-L220

Reply:
@balloob any additional changes needed? Any advice on how to hit that code block that checks whether the device has already been setup?

Reply:
set it up twice?

Reply:
As it is, the test calls

```python
assert await async_setup_component(hass, DOMAIN, CONFIG_ANDROIDTV_ADB_SERVER) 
```

twice, but the coverage shows that block as missed.